### PR TITLE
Adding essential TTT2 items

### PR DESCRIPTION
**Pull request by Alf21**

### CODE REVIEW

File: gamemodes/terrortown/gamemode/shared/lang/english.lua [hunk 1]
```diff
@@ -1278,3 +1278,21 @@ L.door_auto_closes = "This door closes automatically"
...
+L.item_no_fire_damage = "No Fire Damage"
+L.item_no_fire_damage_desc = [[Makes you immune to fire damage.]]
+L.item_no_hazard_damage = "No Hazard Damage"
+L.item_no_hazard_damage_desc = [[Makes you immune to hazard damage such as posion, radiation and acid.]]
+L.item_no_energy_damage = "No Energy Damage"
+L.item_no_energy_damage_desc = [[Makes you immune to energy damage such as lasers, plasma and lightning.]]
+L.item_no_prop_damage = "No Prop Damage"
+L.item_no_prop_damage_desc = [[Makes you immune to prop damage.]]
+L.item_no_drown_damage = "No Drowning Damage"
+L.item_no_drown_damage_desc = [[Makes you immune to drowning damage.]]
```
Comment (lebroomer) [inline gamemodes/terrortown/gamemode/shared/lang/english.lua:1298]: drowning or drown damage?

Comment (TimGoll) [reply gamemodes/terrortown/gamemode/shared/lang/english.lua:1298]: I think `drowing` sounds better

Comment (lebroomer) [reply gamemodes/terrortown/gamemode/shared/lang/english.lua:1298]: It is a little bit inconsistent then since all variable names use ```drown_damage``` .

Comment (TimGoll) [reply gamemodes/terrortown/gamemode/shared/lang/english.lua:1298]: hm, I could rename the internal stuff as well, yes

**Pull request merged**